agenda-webapps/src/main/webapp/vue-app/agenda-common/components/event/view/AgendaEventDetails.vue

```diff
@@ -195,141 +69,44 @@ export default {
-    },
-    ownerDisplayName() {
-      return this.ownerProfile && (this.ownerProfile.displayName || this.ownerProfile.fullname || this.ownerProfile.fullName);
-    },
-    sameDayDates() {
-      return this.event.startDate && this.event.endDate && this.$agendaUtils.areDatesOnSameDay(this.event.startDate, this.event.endDate);
-    },
-    canAddReminders() {
-      if (!this.isAttendee) {
-        return false;
-      }
-      const currentUserResponse = this.event.attendees.find(attendee => attendee && attendee.identity.remoteId ===  eXo.env.portal.userName);
-      return currentUserResponse && (currentUserResponse.response === 'ACCEPTED' || currentUserResponse.response === 'TENTATIVE');
-    },
-    acceptedResponsesCount() {
-      if (!this.event || !this.event.attendees || !this.event.attendees.length) {
-        return 0;
-      }
-      return this.event.attendees.filter(attendee => attendee && attendee.response === 'ACCEPTED').length;
-    },
-    refusedResponsesCount() {
-      if (!this.event || !this.event.attendees || !this.event.attendees.length) {
-        return 0;
-      }
-      return this.event.attendees.filter(attendee => attendee && attendee.response === 'DECLINED').length;
-    },
-    tentativeResponsesCount() {
-      if (!this.event || !this.event.attendees || !this.event.attendees.length) {
-        return 0;
-      }
-      return this.event.attendees.filter(attendee => attendee && attendee.response === 'TENTATIVE').length;
-    },
-    needsActionResponsesCount() {
-      if (!this.event || !this.event.attendees || !this.event.attendees.length) {
-        return 0;
-      }
-      return this.event.attendees.filter(attendee => attendee && attendee.response === 'NEEDS_ACTION').length;
-    },
-    attendeesResponsesTitle() {
-      return this.$t('agenda.attendeesResponsesOverview', {
-        0: this.acceptedResponsesCount,
-        1: this.refusedResponsesCount,
-        2: this.needsActionResponsesCount,
-        3: this.tentativeResponsesCount,
-      });
-    },
   },
-  created() {
-    this.timeZoneName = this.$agendaUtils.getTimeZoneNameFromTimeZoneId(this.$agendaUtils.USER_TIMEZONE_ID);
-  },
-  methods: {
-    closeDialog() {
-      this.$emit('close');
+  methods:{
+    deleteEvent() {
+      this.$eventService.updateEventFields(this.event.id, {
+        status :'CANCELLED'
+      }, false, true)
+        .then(() => {
+          this.$root.$emit('agenda-refresh', this.event);
+          this.$root.$emit('event-canceled', this.event);
+          this.$emit('close');
+        });
     },
     deleteConfirmDialog() {
-      this.$refs.deleteConfirmDialog.open();
-    },
-    deleteEvent() {
-      this.$eventService.deleteEvent(this.event.id)
-        .then(() => this.$root.$emit('agenda-event-deleted', this.event));
+      if (!this.isDatePoll) {
+        this.$refs.deleteConfirmDialog.open();
+      } else {
+        this.$eventService.updateEventFields(this.event.id, {
```

Comment:
you can reuse `this.deleteEvent()` here and in `deleteEvent()` method you choose to refresh the agenda or not by testing on `event.status`.